src/mount/direntry_cache.h

```diff
@@ -573,17 +612,49 @@ class DirEntryCache {
 		return current_time_;
 	}
 
+	/// String representation of the complete cache. Useful for debugging.
```

Comment:
Consider adding pre-processor directive like ```#ifdef DEBUG``` if this function is only needed for debugging to avoid adding this symbols to Release binaries.

Reply:
Will be done in later changes.